### PR TITLE
[1LP][RFR][NOTEST] Removed already automated manual tests related to EC2 provider.

### PR DESCRIPTION
**Pull request by sbulage**
## Purpose or Intent

- [X] Removed manual tests related to EC2 and vSphere 6 provider.

- [X] Moved EC2 SmartState Agent tracker test is moved to main SSA file which has to verified manually.

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 1]
```diff
@@ -989,3 +989,19 @@ def test_ssa_multiple_vms(ssa_multiple_vms, soft_assert, appliance, compare_linu
             compare_linux_vm_data(ssa_vm)
         else:
             compare_windows_vm_data(ssa_vm)
+
+
+@pytest.mark.manual("manualonly")
+def test_ssa_vm_ec2_agent_tracker():
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: Missing: ```@pytest.mark.meta(coverage=[1557452])```

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 2]
```diff
@@ -989,3 +989,19 @@ def test_ssa_multiple_vms(ssa_multiple_vms, soft_assert, appliance, compare_linu
...
+    Bugzilla:
+        1557452
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.9
+        tags: smartstate
+    """
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: - startsin can change to **startsin: 5.10**
- Can you please add **teststeps** and **expectedResults** as this is the ```manualonly``` test case

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py (hunk 1 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: @ganeshhubale I have just moved test from one file to another, have plan to automate it.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py (hunk 2 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: okay

**Pull request merged**